### PR TITLE
D2BotMuleLog.dbj - fix for repetitive login on the same account

### PR DESCRIPTION
**Pull request by mf022**

### CODE REVIEW

File: d2bs/kolbot/D2BotMuleLog.dbj [hunk 1]
```diff
@@ -447,7 +440,8 @@ MainSwitch:
 
 		// last char in acc = trigger next acc
 		if (!charList.length) {
-			nextAcc = true;
+			accounts.shift(); // remove current account from the list
```
Comment (jaenster) [inline d2bs/kolbot/D2BotMuleLog.dbj]: Didnt you shift the account already at line 298?

File: d2bs/kolbot/D2BotMuleLog.dbj [hunk 2]
```diff
@@ -624,7 +618,8 @@ MainSwitch:
 		}
 
 		if (!locationTimeout(5000, location)) {
-			nextAcc = true; // Go to next account
+			accounts.shift(); // remove current account from the list
```
Comment (jaenster) [inline d2bs/kolbot/D2BotMuleLog.dbj]: Didnt you shift the account already at line 298?

**Comment by mf022** [comment]
> At line 318 and 323 you might need to remove the current charlist, as you cant login on the account.

those 2 lines:
```
			accounts.shift(); // remove current account from the list
			chars.shift();
```
were included in default version in nextAcc loop, located at the beginning of case 18
```
		if (nextAcc) {
			accounts.shift();
			chars.shift();

			nextAcc = false;
		}
```
so all I did was to erase that variable, and to replace it with its default 2 lines, in 2 places

---------------
the addition was only to remove the account from the account list (line 298), after you log on that account.

**Comment by jaenster** [comment]
> 
> 
> > At line 318 and 323 you might need to remove the current charlist, as you cant login on the account.
> 
> those 2 lines:
> 
> ```
> 			accounts.shift(); // remove current account from the list
> 			chars.shift();
> ```
> 
> were included in default version in nextAcc loop, located at the beginning of case 18
> 
> ```
> 		if (nextAcc) {
> 			accounts.shift();
> 			chars.shift();
> 
> 			nextAcc = false;
> 		}
> ```
> 
> so all I did was to erase that variable, and to replace it with its default 2 lines, in 2 places
> 
> the addition was only to remove the account from the account list (line 298), after you log on that account.

Good, as long it wasn't unintended its all fine, great to see development!

**Comment by jaenster** [comment]
@noah- this is a relatively small change, i don't know what is holding it up to merge this?

**Pull request merged**